Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h

```diff
@@ -0,0 +1,157 @@
+/*
+ * All or portions of this file Copyright (c) Amazon.com, Inc. or its affiliates or
+ * its licensors.
+ *
+ * For complete copyright and license terms please see the LICENSE at the root of this
+ * distribution (the "License"). All use of this software is governed by the License,
+ * or, if provided, by the license below or the license accompanying this file. Do not
+ * remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
```

Comment:
The license needs to be changed for this to be okay to commit to the repo.

Reply:
I'm seeing the same problem on all these files.  Do we need to rebase on top of development before trying again?

Reply:
Would a rebase fix files that are being added in this PR?

Reply:
Yes, if you rebase it on top of upstream/development (assuming you have upstream configured).

Reply:
Fixed